javafx-archetype-fxml/src/main/resources/archetype-resources/pom.xml

```diff
@@ -8,8 +8,6 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <javafx.version>12-ea+13</javafx.version>
```

Comment:
You could still keep `${javafx.version}`, as it makes sense if you have several JavaFX modules or a multi-modular project.
I'll keep `<javafx.version>${javafx-version}</javafx.version>` for the properties, and `<version>${javafx.version}</version>` for the dependencies

Reply:
I do not understand. Each of the archetype, have their own `javafx-version` property. How is it advantageous to keep a property?

Reply:
The archetype produces a pom, then the user will modify it as required as the project grows. I'm just saying that having the properties block sometimes it is convenient as the dependencies or number of subprojects grow. In this case the archetype will be producing a property for the JavaFX dependencies instead of fixed value.

Reply:
Although the idea makes sense, it appears to me that we might be over-thinking in this particular case. These archetypes are meant for users to create a simple getting started project for JavaFX using Maven. Once the projects get complex, the users will need to make necessary changes to the project to simplify the build process (including moving javafx version to properties). 

This is just my point of view and it might be different for others. I will let others weigh in before making a change.

Reply:
Ok, keep the PR change, it's fine by me.